examples/decoding/plot_decoding_xdawn_eeg.py

```diff
@@ -62,22 +62,20 @@
                 add_eeg_ref=False, verbose=False)
 
 # Create classification pipeline
-clf = make_pipeline(Xdawn(n_components=3),
-                    EpochsVectorizer(),
+e = EpochsVectorizer()
```

Comment:
Try not to use single letters (except `X` and `y `), it can interact with the debugger.

If the instance of `EpochsVectorizer` is only called once, maybe do `X, y = EpochsVectorizer().fit_transform(epochs)`